packages/admin/admin/src/index.ts

```diff
@@ -196,3 +196,5 @@ export { RouterTabsClassKey } from "./tabs/RouterTabs.styles";
 export { Tab, Tabs, TabsProps } from "./tabs/Tabs";
 export { TabsClassKey } from "./tabs/Tabs.styles";
 export { TabScrollButton, TabScrollButtonClassKey, TabScrollButtonProps } from "./tabs/TabScrollButton";
+export { ContentTranslationServiceProvider } from "./translator/config/ContentTranslationServiceProvider";
```

Comment:
Maybe move the files into `src/contentTranslation`? No need for config subfolder though